GitHub-Test-DHprojects/tefftGitTest4.md

```diff
@@ -0,0 +1,8 @@
+## What research question(s) does this project investigate?
+
+They wanted to see the usage of vulgar language as well as who used it in the screenplays, and in what context.
+
+   
+
+## What visualizations or features stand out about this project and why are they interesting? What are they showing?
+The graphs were very useful in showing the distribution of words spoken, and the network graph was also interesting, but without any way to zoom in/interact with it, it was very difficult to see what the connections were and what they meant.
```

Comment:
The use of dynamic visuals really adds to the experience. I use Power BI a lot and one of the outstanding features for me is the dynamic filters & clickability of graphs. However, network graphs are difficult to execute, on Power Bi in particular.